Reach/Payment/Model/ResourceModel/Carrier/CsvHsCode/CSV/RowParser.php

```diff
@@ -82,8 +85,25 @@ private function getHsCode(array $rowData, ColumnResolver $columnResolver)
     {
         $hsCode = $columnResolver->getColumnValue(ColumnResolver::COLUMN_HSCODE, $rowData);
         if ($hsCode === '') {
-            $hsCode = '*';
+            $hsCode = null;
         }
         return $hsCode;
     }
+
+    /**
+     * @param array $rowData
+     * @param ColumnResolver $columnResolver
+     * @return int|string
+     * @throws ColumnNotFoundException
+     */
+    private function getCountryOfOrigin(array $rowData, ColumnResolver $columnResolver)
+    {
+        $countryOfOrigin = $columnResolver->getColumnValue(ColumnResolver::COLUMN_COUNTRYOFORIGIN, $rowData);
+        if (strlen($countryOfOrigin) != 2) {
+            $countryOfOrigin = null;
```

Comment:
is there a way to return or log an error in this case? So that the merchant can know what happened at least?